process/heartbeat/interceptedHeartbeat.go

```diff
@@ -68,6 +67,11 @@ func createHeartbeat(marshalizer marshal.Marshalizer, buff []byte) (*heartbeat.H
 	if err != nil {
 		return nil, err
 	}
+	payload := &heartbeat.Payload{}
```

Comment:
maybe save this payload instance in the interceptedHeartbeat pointer? We almost certain will need it